tasks/eva_3004/eva3004/src/uk/ac/ebi/eva/eva3004/EVACursor.groovy

```diff
@@ -12,21 +12,22 @@ import org.springframework.retry.annotation.Retryable
 class EVACursor<T> implements Iterable<T> {
     CriteriaDefinition filterCriteria
     MongoTemplate mongoTemplate
+    final String collectionName
     final Class<T> collectionClass
     final int pageSize
     MongoCursor<Document> resultIterator
 
     EVACursor() {}
 
-    EVACursor(CriteriaDefinition filterCriteria, MongoTemplate mongoTemplate, Class<T> collectionClass, int pageSize = 1000) {
+    EVACursor(CriteriaDefinition filterCriteria, MongoTemplate mongoTemplate, Class<T> collectionClass, int pageSize = 1000, String collectionName = null) {
```

Comment:
Enable usage of EVACursor on collection names other than the standard ones (like SVE, dbsnpSVE etc.,)..